docs/sources/developers/plugins/data-frames.md

```diff
@@ -26,45 +24,47 @@ interface DataFrame {
 }
 ```
 
-In essence, a data frame is a collection of _fields_, where each field corresponds to a column. Each field, in turn, consists of a collection of values, along with meta information, such as the data type of those values.
+### Data frame fields
+
+In essence, a data frame is a collection of _fields_, where each field corresponds to a column. Each field, in turn, consists of a collection of values, along with metadata, such as the data type of those values.
 
 ```ts
 interface Field {
   name: string;
-  // Prometheus like Labels / Tags
+  // Prometheus-like Labels / Tags
   labels?: Record<string, string>;
 
   // For example string, number, time (or more specific primitives in the backend)
   type: FieldType;
   // Array of values all of the same type
   values: Vector<T>;
 
-  // Optional display data for the field (e.g. unit, name over-ride, etc)
+  // Optional display data for the field (e.g. unit, name override, etc.)
   config: FieldConfig;
 }
 ```
 
-Let's look at an example. The table below demonstrates a data frame with two fields, _time_ and _temperature_.
+Let's look at an example. The following table demonstrates a data frame with two fields, _time_ and _temperature_.
 
 | time                | temperature |
 | ------------------- | ----------- |
 | 2020-01-02 03:04:00 | 45.0        |
 | 2020-01-02 03:05:00 | 47.0        |
 | 2020-01-02 03:06:00 | 48.0        |
 
-Each field has three values, and each value in a field must share the same type. In this case, all values in the time field are timestamps, and all values in the temperature field are numbers.
+Each field has three values, and each value in a field must share the same type. In this case, all values in the `time` field are timestamps, and all values in the `temperature` field are numbers.
 
 One restriction on data frames is that all fields in the frame must be of the same length to be a valid data frame.
 
-### Field configuration
+### Field configurations
 
 Each field in a data frame contains optional information about the values in the field, such as units, scaling, and so on.
 
 By adding field configurations to a data frame, Grafana can configure visualizations automatically. For example, you could configure Grafana to automatically set the unit provided by the data source.
 
-## Transformations
+## Data transformations
```

Comment:
Maybe we can link to https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/ and ensure we're still up to date here

Reply:
Good catch, I've commited a fix.